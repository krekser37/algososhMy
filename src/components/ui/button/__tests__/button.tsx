import  renderer  from "react-test-renderer";
import { Button } from "../button";

const buttonText = 'Click me!'

describe('test Button component', ()=>{
    it('render button with text', ()=>{
        const tree = renderer.create(<Button text={buttonText}/>).toJSON()

        expect(tree).toMatchSnapshot()
    })

    it('render disabled with', ()=>{
        const tree = renderer.create(<Button text={buttonText} disabled/>).toJSON()

        expect(tree).toMatchSnapshot()
    })

    
})