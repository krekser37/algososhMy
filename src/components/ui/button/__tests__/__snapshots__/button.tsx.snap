// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Button component render button with text 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  >
    Click me!
  </p>
</button>
`;

exports[`test Button component render button with text 2`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  >
    Click me!
  </p>
</button>
`;

exports[`test Button component render disabled with 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  disabled={true}
  type="button"
>
  <p
    className="text undefined"
  >
    Click me!
  </p>
</button>
`;

exports[`test Button component render disabled with text 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  >
    Click me!
  </p>
</button>
`;
